cpp/src/io/avro/avro_gpu.cu

```diff
@@ -283,17 +329,16 @@ __global__ void __launch_bounds__(num_warps * 32, 2)
                           device_span<string_index_pair const> global_dictionary,
                           uint8_t const* avro_data,
                           uint32_t schema_len,
-                          uint32_t min_row_size,
-                          size_t max_rows,
-                          size_t first_row)
+                          uint32_t min_row_size)
 {
   __shared__ __align__(8) schemadesc_s g_shared_schema[max_shared_schema_len];
   __shared__ __align__(8) block_desc_s blk_g[num_warps];
 
+  bool skipped_row;
```

Comment:
I don't see this used outside of the block with `avro_decode_row`. Can it be declared there?